src/main/java/mat/client/Mat.java

```diff
@@ -649,26 +647,19 @@ private void harpOktaLogout(String redirectTo) {
             @Override
             public void onFailure(Throwable throwable) {
                 logger.log(Level.SEVERE, "HarpService::getHarpUrl -> onFailure " + throwable.getMessage(), throwable);
-                removeOktaTokens();
             }
 
             @Override
             public void onSuccess(String harpUrl) {
                 logger.log(Level.INFO, "HarpService::getHarpUrl -> onSuccess");
-                harpLogout(harpUrl);
                 MatContext.get().getSynchronizationDelegate().setLogOffFlag();
-                MatContext.get().handleSignOut("SIGN_OUT_EVENT", redirectTo);
-                removeOktaTokens();
+                MatContext.get().handleSignOut("SIGN_OUT_EVENT", null);
```

Comment:
if it's set to null, then redirectUrl will not work to Harp Support page

Reply:
Understood, I'll fix it shortly

Reply:
Fixed.